evalml/tests/component_tests/test_arima_regressor.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+import pandas as pd
+import pytest
+from pytest import importorskip
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components import ARIMARegressor
+from evalml.problem_types import ProblemTypes
+
+arima = importorskip('statsmodels.tsa.arima_model', reason='Skipping test because ARIMA not installed')
+
+
+def test_model_family():
+    assert ARIMARegressor.model_family == ModelFamily.ARIMA
+
+
+def test_problem_types():
+    assert set(ARIMARegressor.supported_problem_types) == {ProblemTypes.TIME_SERIES_REGRESSION}
+
+
+def test_feature_importance(ts_data):
+    X, y = ts_data
+    clf = ARIMARegressor()
+    clf.fit(X, y)
+    clf.feature_importance == np.zeros(1)
+
+
+def test_fit_predict_ts_with_Xandy_index(ts_data):
+    X, y = ts_data
+    assert isinstance(X.index, pd.DatetimeIndex)
+    assert isinstance(y.index, pd.DatetimeIndex)
+
+    a_clf = arima.ARIMA(endog=y, exog=X, order=(1, 0, 0), dates=X.index)
+    a_clf.fit(solver='nm')
+    y_pred_a = a_clf.predict(params=(1, 0, 0))
+
+    clf = ARIMARegressor(p=1, d=0, q=0)
+    clf.fit(X=X, y=y)
+    y_pred = clf.predict(X=X, y=y)
+
+    assert (y_pred == y_pred_a).all()
```

Comment:
Do you think this is parameterizable?  (If yes, feel free to come to my #1990 PR and tell me my tests are parameterizable too ;)